webapp/src/components/task_creation_form/TaskCreationForm.vue

```diff
@@ -1,38 +1,886 @@
+                </FormField>
+              </FormLabel>
+            </div>
+          </div>
+        </IconCard>
+
+        <IconCard>
+          <template #title> Training </template>
+
+          <div class="flex flex-row flex-wrap">
+            <FormLabel label="Epochs" type="required">
+              <FormField
+                name="trainingInformation.epochs"
+                placeholder="30"
+                as="input"
+                type="number"
+                min="1"
+              />
+            </FormLabel>
+
+            <FormLabel label="Batch size" type="required">
+              <FormField
+                name="trainingInformation.batchSize"
+                placeholder="10"
+                as="input"
+                type="number"
+                min="1"
+              />
+            </FormLabel>
+
+            <FormLabel
+              label="Fraction of dataset used for validation"
+              type="required"
+            >
+              <FormField
+                name="trainingInformation.validationSplit"
+                placeholder="0.1"
+                as="input"
+                type="number"
+                min="0"
+                step="0.1"
+                max="1"
+              />
+            </FormLabel>
+          </div>
+        </IconCard>
+
+        <IconCard>
+          <template #title> Network </template>
+
+          <div class="flex flex-col">
+            <FormLabel label="Collaborative algorithm" type="required">
+              <FormField
+                name="trainingInformation.scheme"
+                as="select"
+                v-model="scheme"
+                supress-error
+              >
+                <option value="federated">Federated</option>
+                <option value="decentralized">Decentralized</option>
+                <option value="local">Local</option>
+              </FormField>
+            </FormLabel>
+
+            <FormLabel label="Type of aggregation to perform" type="required">
+              <FormField
+                name="trainingInformation.aggregationStrategy"
+                as="select"
+                v-model="aggregationStrategy"
+              >
+                <option value="mean">Mean</option>
+                <option v-if="scheme === 'decentralized'" value="secure">
+                  Secure
+                </option>
+              </FormField>
+
+              <FormLabel
+                label="Maximum absolute value over all the weights"
+                v-show="aggregationStrategy === 'secure'"
+              >
+                <FormField
+                  name="trainingInformation.maxShareValue"
+                  placeholder="100"
+                  as="input"
+                  type="number"
+                />
+              </FormLabel>
+            </FormLabel>
+
+            <FormLabel
+              label="Number of epochs before exchanging with the network"
```

Comment:
```suggestion
              label="Number of epochs before aggregating weights"
```
Since we're saying "Type of aggregation" in the previous field we might as well re-use the same language I think?